base_address_parish/views/res_partner_view.xml

```diff
@@ -0,0 +1,76 @@
+<odoo>
+
+    <record id="parish_view_partner_form" model="ir.ui.view">
+      <field name="name">parish.partner.view</field>
+      <field name="model">res.partner</field>
+      <field name="inherit_id" ref="base.view_partner_form" />
+      <field name="arch" type="xml">
+        <field name="city" position="attributes">
+          <attribute name="invisible">1</attribute>          
+        </field>
+        <field name="city" position="after">
+          <field name="city_id" placeholder="City" class="o_address_city" attrs="{'readonly': [('type', '=', 'contact'),('parent_id', '!=', False)]}"/>
+        </field>
```

Comment:
Because this??

Please see https://github.com/odoo/odoo/blob/15.0/addons/base_address_city/models/res_partner.py#L42-L60

Reply:
Because in base_address_city don't have view for show city_id, so, I show beacuse parish it's depending of city_id

Reply:
> Because in base_address_city don't have view for show city_id, so, I show beacuse parish it's depending of city_id

Of course, fields are added to form view from python code, see link

Reply:
see also https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/models/res_partner.py#L49-L67